pkg/mark/testdata/quotes.md

```diff
@@ -29,3 +29,39 @@
 ## Simple Blockquote
 
 > This paragraph is a simple blockquote
+
+## GH Alerts Heading
+
+### Note Type Alert Heading
+
+> [!NOTE]
```

Comment:
Can you add some negative test cases which should not be rendered?

something like
`> [[!NOTE]`
or 
`> [!NOTE`
or 
`> [NOTE]`

Reply:
and something like

```
> **NOTES:**
>
> 1. Note number one
> 1. Note number two
>
>> a
>> b
>
> **Warn (Should not be picked as blockquote type)**
```
This testcase (the warn one) with Github Alerts